### DISCUSSION TITLE
How to use logs with Linkerd, the command in doc is deleted

### DISCUSSION THREAD

**Discussion by denglisch** [linkerd/linkerd2#5790] [General]
Hello I'm searching for the right way to see logs with linkerd. I have version 2.9.3 installed. If I ask the Doku: https://linkerd.io/2/reference/cli/logs/ there is a command that was deleted.
If I look in the CHANGES.md I see at point edge-20.11.2:
- Removed the get and logs command from the CLI

But I can't find an issue or a doku why this is happed and what is the new way for this task.

**Comment by kleimkuhler** [comment]
Hey @denglisch. The `linkerd logs` command was removed because of it's overlap with `kubectl logs`. If you'd like to view the logs for the Linkerd destination container, you'd use the following:

```
$ kubectl logs -n linkerd -f linkerd-destination-... destination
```

If you want to view the logs for the proxy, change the container name:


```
$ kubectl logs -n linkerd -f linkerd-destination-... linkerd-proxy
```

This holds for any of your application pods that are injected. If you want that logs for a pod's proxy, you would use the `kubectl logs` command, specifying the pod's `linkerd-proxy` container.

Let me know if this helps out of if you have any other questions.

**Reply by denglisch** [reply]
thank you for the fast answer. I'll try It out and will give feedback soon.

**Comment by denglisch** [comment]
i have one more question, if I have more then one instance of an app, did I have to know witch instance make an error or can I see all logs (or is there an other solution)?

**Reply by kleimkuhler** [reply]
In that scenario I would recommend using Linkerd. There are several tools available that can help figure out where errors are originating from. I'd start with the `stat` command to get an idea about traffic to your resources. From there consider referencing the [debugging](https://linkerd.io/2/tasks/books/) task in the docs about narrowing things down further.
